src/app/lib/preventChangeOnNumberInputScroll.ts

```diff
@@ -0,0 +1,19 @@
+import { TextInputProps } from 'grommet/es6/components/TextInput'
+
+export const preventChangeOnNumberInputScroll = (): Pick<TextInputProps, 'onWheel'> => {
+  return {
+    onWheel: e => {
+      const target = e.target as HTMLElement
+
+      // Prevents input value change
+      target.blur()
+      // Prevents scrolling
+      e.stopPropagation()
+
+      // Focus after blur
+      setTimeout(() => {
+        target.focus()
+      }, 0)
```

Comment:
Unintended sideffect: input becomes focused when scrolling over it

Reply:
So I checked, wheel event does not get triggered on mobile. So I added an implementation of global listener here https://github.com/oasisprotocol/oasis-wallet-web/pull/1964.

I would expect the input[type="field"] to not lose focus, while scrolling on it. But you can try global listener implementation with `preventDefault()` that would prevent scrolling at all - if that implementation is more intuitive for you. Or we could simply remove `target.focus()`, but that way it would always lose focus, when scrolling over the input[type="number"] field.

Reply:
Ok, this has been resolved now, if input is focused, it will keep focus. Otherwise focus won't be captured on un-focused field. But feel like implementation is far from ideal, and I think I would rather go with https://github.com/oasisprotocol/oasis-wallet-web/pull/1964 for simplicity sake - even though it does not work the same as this implementation.